openedx/core/djangoapps/user_api/legacy_urls.py

```diff
@@ -12,6 +14,7 @@
 USER_API_ROUTER.register(r'user_prefs', user_api_views.UserPreferenceViewSet)
 
 urlpatterns = [
+    re_path(r'^account(?:/settings)?/?$', RedirectView.as_view(url=settings.ACCOUNT_MICROFRONTEND_URL)),
     path('user_api/v1/', include(USER_API_ROUTER.urls)),
```

Comment:
Why you made the suffix  `/settings` in url path as optional?  i

Reply:
Note sure if it's suppose to redierct from `/settings`, AFAIK `lms_base/account` on sumac didn't redirect to account mfe.

Reply:
I made a correction in my earlier comment,  I meant from `lms.base/account`

> Mmm I see what you say. But when using LMS base domain == MFE domain, /account will redirect to the account MFE. Does that make sense? 

when/if (LMS base domain == MFE domain),  then it will a add new diemension to the problem, of which I haven't tried or setup before. Is there a sandbox with such setup?

Reply:
edunext has one with that setup, it has another issues I think but all of them are solved here https://github.com/eduNEXT/tutor-contrib-mfe-extensions

Reply:
Aha gotcha... I think then may be since redirection is at caddy level, it would get redirected before reaching django urls roles. 

Nonetheless, the project provide an intresting solution for the multiesite and MFEs.